### PR TITLE
Add german translation and some small fixes

### PR DESCRIPTION
**Pull request by whot**

### CODE REVIEW

File: po/LINGUAS [hunk 1]
```diff
@@ -1,2 +1,3 @@
 # Language list must be in alphabetical order
 nl
+de
```
Comment (Hjdskes) [inline po/LINGUAS]: Please put this in the right spot.

File: po/nl.po [hunk 2]
```diff
@@ -189,7 +189,7 @@ msgstr "Kan niet verbinden met ratbagd"
 #. Translators: "it" here is ratbagd.
 #: piper/window.py:57
 msgid "Please make sure it is running"
```
Comment (Hjdskes) [inline po/nl.po]: You should update this string as well; in general running `ninja -C build piper-update-po` is the easiest.

Comment (whot) [inline po/nl.po]: that's how I got to the german one. sorry, the software engineer in me refused to run a whole update for a single line change :) Anyway, fixed with your follow-up commits, thanks.

**Comment by whot** [comment]
commited as series up to a0753fdd738fd3c6671f8c5be32ad09fd6cd5029

**Pull request closed without merging**